### PR TITLE
Feature/weather

### PR DESCRIPTION
**Pull request by AlexDunmow**
!weather, returns current BOM temperature & forecast

Also contains fix for build.go, added ":glide up" before go install.

### CODE REVIEW

**Comment by AlexDunmow** [comment]
@nii236, @tcuthbert plz to review

File: responses/weather.go [hunk 1]
```diff
@@ -0,0 +1,373 @@
...
+func Weather(cr *messagerouter.CommandRequest, w messagerouter.ResponseWriter) error {
+	var temperature string
+	var forecast string
+	var wdata W_product
+	var wtemp WTEMP_product
+	var ftp *goftp.FTP
+	var err error
+
+	// For debug messages: goftp.ConnectDbg("ftp.server.com:21")
+	if ftp, err = goftp.Connect("ftp.bom.gov.au:21"); err != nil {
```
Comment (sqweek) [inline responses/weather.go:23]: ftp? that still exists? :P

File: responses/weather.go [hunk 2]
```diff
@@ -0,0 +1,373 @@
...
+
+		for {
+			token, _ := decoder.Token()
+			if token == nil {
+				break
+			}
+			switch se := token.(type) {
+			case xml.StartElement:
+				if se.Name.Local == "product" && se.Name.Space == "" {
+					fmt.Println("DECODING")
```
Comment (sqweek) [inline responses/weather.go]: Does this debug spam need to remain?

File: responses/weather.go [hunk 3]
```diff
@@ -0,0 +1,373 @@
...
+		decoder := xml.NewDecoder(r)
+
+		for {
+			token, _ := decoder.Token()
+			if token == nil {
+				break
+			}
+			switch se := token.(type) {
+			case xml.StartElement:
+				fmt.Println("Decode")
```
Comment (sqweek) [inline responses/weather.go]: Debug spam.

**Review by sqweek — APPROVED** [review]
Looks fine to me.

File: responses/weather.go (hunk 2 above)
Comment (AlexDunmow) [reply responses/weather.go]: GOOD point!

**Pull request merged**